spec/factories/tasks.rb

```diff
@@ -27,5 +27,14 @@
     description { Faker::Lorem.paragraph }
 
     project
+
+    trait :with_files do
+      after(:build) do |user|
+        2.times do |n|
+          file = ["image1.jpg", "image2.jpg", "asana.png", "text.txt", "book.pdf"].sample
+          user.files.attach(io: File.open(Rails.root.join('spec', 'factories', 'files', file)), filename: file, content_type: ['image/png', 'image/jpg', 'text/plain', 'application/pdf'])
```

Comment:
why you have array of content types? simplify and reduce formats or for each file put appropriate content type